frontend/smart-kickers-game/src/App.js

```diff
@@ -31,13 +43,36 @@ function App() {
     });
   }
 
+  function handleEndGame() {
+    setToggleGameScore(!toggleGameScore);
+  }
+
   return (
     <>
       <h1>Smart Kickers</h1>
-      <GameResults blueScore={blueScore} whiteScore={whiteScore} />
-      <center>
-        <Button onClick={() => handleResetGame()}>Reset game</Button>
-      </center>
+      {toggleGameScore === false ? (
+        <>
+          {<GameResults blueScore={blueScore} whiteScore={whiteScore} />}{' '}
+          {
+            <>
+              <center className="game-ending-buttons">
+                <Button onClick={() => handleResetGame()}>Reset game</Button>
+                <br />
+                <Button
+                  onClick={() => {
+                    setFinalScores({ blue: blueScore, white: whiteScore });
+                    handleEndGame();
+                  }}
+                >
+                  End game
+                </Button>
+              </center>
+            </>
+          }{' '}
+        </>
```

Comment:
You can create a separate component for that.

Reply:
Done. Created a component named `CurrentGameplay.js`